packages/website_v2/src/components/GovernancePage/GrantsCommittee/GrantsCommittee.scss

```
@@ -0,0 +1,112 @@
+.tw__grantsCommittee {
+  position: relative;
+  background: linear-gradient(89.84deg, #101669 0.14%, #c6bfff 99.42%);
+  color: #eeecff;
+
+  .tw__grantsCommittee__background-desktop {
+    position: absolute;
+    left: 45%;
+    right: 0px;
+    top: 5%;
+    bottom: 5%;
+
+    @media screen and (max-width: 1265px) {
+      width: 0px;
+      height: 0px;
+    }
+
+    svg {
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  .tw__grantsCommittee__background-mobile {
+    margin-bottom: 40px;
+
+    @media screen and (min-width: 1266px) {
+      display: none;
+    }
+
+    svg {
+      width: 210px;
+      height: 241px;
+    }
+  }
+
+  .tw__grantsCommittee__container {
+    padding: 40px 40px 100px 40px;
+
+    @media screen and (max-width: 1265px) {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      padding: 60px 0px 60px 0px;
```

It can be shortened to

```suggestion
      padding: 60px 0px;
```